fix_breakline.py

```
@@ -0,0 +1,43 @@
+import sys
+
+with open('src/scripts/scenes/MainScene.js', 'r') as f:
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)